package.json

```diff
@@ -176,7 +184,7 @@
   },
   "syncpack": {
     "prod": true,
-    "dev": true,
+    "dev": false,
```

Comment:
I turned off checks for devDeps. This tool wont work with what we agreed upon 
-> single version policy === exact versions
-> published package deps === caret versions

Reply:
this works:

```
   {
        "packages": [
          "fluent-ui-react-repo"
        ],
        "dependencies": [
          "tslib",
          "webpack-bundle-analyzer",
          "chalk"
        ]
      },
```

although I don't think this is what we want, as it will allow further misalignments between versions.

What would be probably needed as a feature for syncpack is the ability to specify devDependencies similarly to how dependencies work.